api/include/opentelemetry/trace/default_span.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "opentelemetry/common/attribute_value.h"
+#include "opentelemetry/trace/canonical_code.h"
+#include "opentelemetry/trace/span.h"
+#include "opentelemetry/trace/span_context.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace trace
+{
+class DefaultSpan : public Span
+{
```

Comment:
nitpik: Can we use `override` keyword for the function which are virtually defined in `Span` base class. Will make this class more readable.

Reply:
I just did it.